CMakeLists.txt

```diff
@@ -105,6 +105,17 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL "Generic")
     endif()
 endif()
 
+# Check external transport.
+SET(EXTERNAL_TRANSPORT_HEADER ""   CACHE PATH "External transports header file")
+SET(EXTERNAL_TRANSPORT ""          CACHE PATH "External transports code file")
```

Comment:
Please use lowercase for CMake functions.

Reply:
Done